src/Symfony/Component/Serializer/NameConverter/NameConverterInterface.php

```diff
@@ -21,16 +21,20 @@
     /**
      * Converts a property name to its normalized value.
      *
-     * @param string $propertyName
+     * @param string|object $class
+     * @param string        $propertyName
+     *
      * @return string
      */
-    public function normalize($propertyName);
+    public function normalize($class, $propertyName);
```

Comment:
According to the BC promise, you'd need to make `$class` the second argument and make it optional. That's not ideal, but I don't think there's another way